main/zoe/api/contract-support/price-authority.md

```diff
@@ -136,3 +136,68 @@ const quote = E(pAuthority).quoteWhenLTE(
     amountMath.make(brands.Out, 974n),
   );
 ```
+## `MutableQuote` object
+
+**tyg todo: When do you want to use a mutable quote vs. a regular quote, and vice versa?**
+
+There are four *mutable quote* methods, which return a `MutableQuote` object with the methods:
+- `cancel(e)`
+  - `e``{ String }`
+  - Causes the `Promise` to reject with the message `e`.  **tyg todo: How is the message used; where does it go?**
```

Comment:
Hmm. There's nothing in our doc about how to use promises, other than a reference to javascript.info's docs. That'll have to do, I guess. When someone makes use of the promise with a `.then()`, they'll be notified of rejections. That's where it is received.